### PR TITLE
Add headers to folders

### PR DESCRIPTION
**Pull request by yirenlu92**
- Add headers to non-ML examples that are missing them

### CODE REVIEW

**Review by kning — APPROVED** [review]

File: 01_getting_started/generators.py [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# # Run a generator function on Modal
+
+# This example shows how you can run a generator function on Modal. We define a
+# function that yields values and then call it with the [`remote_gen`](https://modal.com/docs/reference/modal.Function#remote_gen) method. The
```
Comment (charlesfrye) [inline 01_getting_started/generators.py:4]:
```suggestion
# function that `yields` values and then call it with the [`remote_gen`](https://modal.com/docs/reference/modal.Function#remote_gen) method. The
```

File: 05_scheduling/schedule_simple.py [hunk 2]
```diff
@@ -1,6 +1,21 @@
 # ---
 # cmd: ["python", "-m", "05_scheduling.schedule_simple"]
 # ---
+
+# # Scheduling remote jobs
+
+# This example shows how you can schedule remote jobs on Modal.
+# You can do this either with:
+#
+# - [modal.Period](https://modal.com/docs/reference/modal.Period) - a time interval between function calls.
```
Comment (charlesfrye) [inline 05_scheduling/schedule_simple.py:10]:
```suggestion
# - [`modal.Period`](https://modal.com/docs/reference/modal.Period) - a time interval between function calls.
```

File: 05_scheduling/schedule_simple.py [hunk 3]
```diff
@@ -1,6 +1,21 @@
...
+# - [modal.Cron](https://modal.com/docs/reference/modal.Cron) - a cron expression to specify the schedule.
```
Comment (charlesfrye) [inline 05_scheduling/schedule_simple.py:11]:
```suggestion
# - [`modal.Cron`](https://modal.com/docs/reference/modal.Cron) - a cron expression to specify the schedule.
```

File: 07_web_endpoints/count_faces.py [hunk 4]
```diff
@@ -2,6 +2,16 @@
 # lambda-test: false
 # ---
 
+# Run opencv face detection on an image
```
Comment (charlesfrye) [inline 07_web_endpoints/count_faces.py:5]:
```suggestion
# Run OpenCV face detection on an image
```

File: 07_web_endpoints/fastapi_app.py [hunk 5]
```diff
@@ -2,6 +2,11 @@
 # lambda-test: false
 # ---
 
+# # Deploy FastAPI app with Modal
+
+# This example shows how you can deploy a [FastAPI](https://fastapi.tiangolo.com/) app with Modal.
+# You can serve any app written in an ASGI or WSGI-compatible web framework (like FastAPI) on Modal.
```
Comment (charlesfrye) [inline 07_web_endpoints/fastapi_app.py:8]:
```suggestion
# You can serve any app written in an ASGI-compatible web framework (like FastAPI) using this pattern or you can server WSGI-compatible frameworks like Flask with [`wsgi_app`](https://modal.com/docs/guide/webhooks#wsgi).
```

File: 07_web_endpoints/fasthtml_app.py [hunk 6]
```diff
@@ -2,6 +2,13 @@
 # deploy: true
 # cmd: ["modal", "serve", "07_web_endpoints/fasthtml_app.py"]
 # ---
+
+# # Deploy FastHTML app with Modal
```
Comment (charlesfrye) [inline 07_web_endpoints/fasthtml_app.py:6]:
```suggestion
# # Deploy a FastHTML app with Modal
```

**Review by charlesfrye — COMMENTED** [review]
accidentally started a review. sorry!

File: 07_web_endpoints/fasthtml_app.py [hunk 7]
```diff
@@ -2,6 +2,13 @@
 # deploy: true
 # cmd: ["modal", "serve", "07_web_endpoints/fasthtml_app.py"]
 # ---
+
+# # Deploy a FastHTML app with Modal
+
+# This example shows how you can deploy a FastHTML app with Modal.
+# [FastHTML](https://www.fastht.ml/) is a Python library that allows you to create HTML elements using Python code.
```
Comment (charlesfrye) [inline 07_web_endpoints/fasthtml_app.py:9]:
```suggestion
# [FastHTML](https://www.fastht.ml/) is a Python library that allows you to create entire web applications using only Python.
```

File: 07_web_endpoints/flask_app.py [hunk 8]
```diff
@@ -2,6 +2,11 @@
 # lambda-test: false
 # ---
 
+# # Deploy Flask app with Modal
+
+# This example shows how you can deploy a [Flask](https://flask.palletsprojects.com/en/3.0.x/) app with Modal.
+# You can serve any app written in an ASGI or WSGI-compatible web framework (like Flask) on Modal.
```
Comment (charlesfrye) [inline 07_web_endpoints/flask_app.py:8]:
```suggestion
# You can serve any app written in a WSGI-compatible web framework (like Flask) on Modal with this pattern. You can serve an app written in an ASGI-compatible framework, like FastAPI, with [`asgi_app`](https://modal.com/docs/guide/webhooks#asgi).
```

File: 08_advanced/parallel_execution.py [hunk 9]
```diff
@@ -1,3 +1,9 @@
+# # Parallel Execution on Modal
```
Comment (charlesfrye) [inline 08_advanced/parallel_execution.py:1]:
```suggestion
# # Parallel execution on Modal with `spawn`
```

File: misc/lmdeploy_oai_compatible.py [hunk 10]
```diff
@@ -1,3 +1,7 @@
+# # Deploy a model with lmdeploy
```
Comment (charlesfrye) [inline misc/lmdeploy_oai_compatible.py:1]:
```suggestion
# # Deploy a model with `lmdeploy`
```

**Comment by charlesfrye** [comment]
nice work Ren!

**Pull request merged**